siliconcompiler/apps/sc_show.py

```diff
@@ -55,10 +55,19 @@ def main():
         if ext in default_input_map:
             input_map[ext] = default_input_map[ext]
 
-    chip.create_cmdline(progname,
-                        switchlist=['-design', '-input', '-loglevel', '-cfg'],
-                        description=description,
-                        input_map=input_map)
+    extension_arg = {
+        'metavar': '<ext>',
+        'help': 'Specify the extention of the file to show'
```

Comment:
typo: extention -> extension

Reply:
Also, I think it could be helpful to specify default behavior/explicitly mention that this flag is optional.